### ISSUE TITLE
[Bug]: Issue with doggy talents Next

### ISSUE THREAD

**Issue by Esseit** [Fuzss/moblassos#7]
### Mod Loader (Required)

Forge

### Minecraft Version(s) (Required)

1.20.1

### Mod Version(s) (Required)

8.0.1

### Minimal Setup (Required)

Yes

### Notes (Required)

When capturing a dog from DTN in a lasso of any kind, dog is then stuck in the lasso and unable to be deployed. Along side this, several right click options then become blocked until death or reloading the world.

### latest.log (Required)

https://pastebin.com/6AUtiC2r

**Comment by Farqiplier** [comment]
I also had this issue with DTN, both with this mod and another mod called mob catcher. As far as I know i got no way to place my dogs back so I guess they're gone forever, sadly. (If someone knows how to save my dogs please tell me how i can get them out)

**Comment by MyDarlingirene** [comment]
I came looking for this exact reason. Since I know that that info alone isnt enough, I was using a diamond lasso with the doggy talent next mod. The dog was already named. Sad to hear that he's trapped until the bug is fixed. I hope it gets fixed in the next patch.

**Comment by scribe-of-days** [comment]
I can confirm this is an issue. Once captured it's impossible to deploy a DTN dog from a lasso. If the lasso breaks, then it basically softlocks your character in a state of limbo--I was able to fix it by deleting the lasso from my NBT data, and then using the dog bed to bring my dog back.

**Comment by Fuzss** [comment]
Yeah, there is something fishy going on here. Ideally configure this mod so that dogs cannot be captured.
